commit f797e273c3b535f22691759692b7894f32c9b69d (refs/changes/36/1736/4)
Author: Kody A Kantor <kody@kkantor.com>
Date:   2018-04-27T14:39:31+00:00 (1 year, 5 months ago)
    
    MANTA-2980 electric-moray should report moray shard for operations like 'putObject'

diff --git a/lib/moray.js b/lib/moray.js
index fe8050c..491f308 100644
--- a/lib/moray.js
+++ b/lib/moray.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 //
@@ -433,7 +433,7 @@ function put(options, cb) {
         headers: opts.headers
     }, 'Moray.' + op + ': entered');
 
-    client.putObject(bucket, key, md, opts, function (err) {
+    client.putObject(bucket, key, md, opts, function (err, data) {
         if (err) {
             log.debug({
                 err: err,
@@ -454,7 +454,7 @@ function put(options, cb) {
                 key: key,
                 requestId: opts.requestId
             }, 'Moray.' + op + ': done');
-            cb(null, md);
+            cb(null, md, data);
         }
     });
 }
